src/main/java/org/jitsi/videobridge/IceUdpTransportManager.java

```diff
@@ -391,65 +396,67 @@ public boolean addChannel(Channel channel)
         if (closed)
             return false;
 
-        if (channel instanceof SctpConnection
-                && sctpConnection != null
-                && sctpConnection != channel)
-        {
-            logd("Not adding a second SctpConnection to TransportManager.");
-            return false;
-        }
-
-        if (!super.addChannel(channel))
-            return false;
-
-        if (channel instanceof SctpConnection)
+        synchronized (channelForDtlsLock)
```

Comment:
The diff here is actually pretty small - the only thing that changes inside the synchronize block in this commit is whitespace.